run_tests.py

```
@@ -73,47 +74,21 @@ def main():
-        "--no-coverage",
-        action="store_true",
-        help="Disable coverage reporting"
-    )
+    parser.add_argument("--no-coverage", action="store_true", help="Disable coverage reporting")
 
-    parser.add_argument(
-        "--html-coverage",
-        action="store_true",
-        help="Generate HTML coverage report"
-    )
+    parser.add_argument("--html-coverage", action="store_true", help="Generate HTML coverage report")
```

Pylint (3.3.7)

[convention] Line too long (101/100)

(C0301)